detect_variants/detect_variants.cwl

```diff
@@ -63,6 +66,9 @@ inputs:
         type: boolean?
     hgvs_annotation:
         type: boolean?
+    filter_annotated_vcf:
```

Comment:
The name of this workflow step is a bit ambiguous. What is being filtered? There are other steps that are hard or soft filtering.

Reply:
how about `cle_indel_filter`